repos/fdbt-site/tests/pages/api/returnService.test.ts

```diff
@@ -0,0 +1,42 @@
+import returnService from '../../../src/pages/api/returnService';
+import { getMockRequestAndResponse, mockRawService } from '../../testData/mockData';
+import * as auroradb from '../../../src/data/auroradb';
+import { TXC_SOURCE_ATTRIBUTE } from '../../../src/constants/attributes';
+
+beforeEach(() => {
+    jest.resetAllMocks();
+
+    jest.spyOn(auroradb, 'getServiceByIdAndDataSource').mockResolvedValue(mockRawService);
+});
+
+describe('service', () => {
+    it('should return 302 redirect to /returnService when there is no body in the request', async () => {
+        const writeHeadMock = jest.fn();
+        const { req, res } = getMockRequestAndResponse({
+            cookieValues: {},
+            body: {},
+            uuid: {},
+            mockWriteHeadFn: writeHeadMock,
+        });
+        await returnService(req, res);
+        expect(writeHeadMock).toBeCalledWith(302, {
+            Location: '/returnService',
+        });
+    });
+
+    it('should return 302 redirect to /direction', async () => {
```

Comment:
It should also set the RETURN_SERVICE_ATTRIBUTE

Reply:
how to check that ... using getSessionAttribute?